package/rpm/README.md

```diff
@@ -0,0 +1,11 @@
+# Corrade RPM Package
+
+You will need to install the following dependencies for build rpms:
+```
+sudo dnf install fedora-packager rpmdevtools gcc
+```
+
+Create user directory tree building RPMs:
+```
+mkdir -p ~/rpmbuild/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
```

Comment:
#125 calls `rpmdev-setuptree`, would that be the equivalent to this?

I suppose it could be called inside `build.sh` directly.